utils/installer/lv-config.example.lua

```diff
@@ -82,6 +82,13 @@ O.lang.go.autoformat = true
 
 -- rust
 O.lang.rust.autoformat = true
+
+-- clang
+O.lang.clang.active = false -- make this true to enable clang support
```

Comment:
You no longer need to set active here..
It will only activate if you:

  1. open a c/cpp file.
  2. Have installed cpp support (LspInstall cpp)

  If you remove this, I'll be happy to approve.